code/modules/client/client_procs.dm

```diff
@@ -651,6 +651,9 @@ GLOBAL_LIST(external_rsc_urls)
 			return FALSE
 		if ("key")
 			return FALSE
+		if("view")
+			change_view(var_value)
+			return FALSE
```

Comment:
This needs to return TRUE, otherwise it says "Your edit was rejected by the object!"